napari/layers/shapes/shapes.py

```diff
@@ -1451,7 +1451,7 @@ def _map_color(self, attribute: str, update_color_mapping: bool = False):
                 # and update_color_mapping==False
                 color_cycle_map = getattr(self, f'{attribute}_color_cycle_map')
                 color_cycle_keys = [*color_cycle_map]
-                props_in_map = np.in1d(color_properties, color_cycle_keys)
+                props_in_map = np.isin(color_properties, color_cycle_keys)
```

Comment:
`in1d` is deprecated and `isin` is the suggested replacement. I'm pretty sure for this use it is a drop-in replacement. I believe `isin` can be slower for certain array dtypes (`object`), but I'm not sure that's an issue here.